Source/engine/size.hpp

```diff
@@ -6,6 +6,20 @@ struct Size {
 	int width;
 	int height;
 
+	Size() = default;
+
+	constexpr Size(int width, int height)
+	    : width(width)
+	    , height(height)
+	{
+	}
+
+	explicit constexpr Size(int size)
+	    : width(size)
+	    , height(size)
+	{
+	}

```

Comment:
Do we finally have a solution the the rectangle vs square problem of programming :o